tests/integration/ie-tests.sh

```diff
@@ -203,18 +203,21 @@ teardown_test_pod() {
 }
 
 run_example_job_in_pod() {
-  SPARK_EXAMPLES_JAR_NAME='spark-examples_2.12-3.3.2.jar'
+  SPARK_EXAMPLES_JAR_NAME='spark-examples_2.12-3.4.0.jar'
 
   PREVIOUS_JOB=$(kubectl get pods | grep driver | tail -n 1 | cut -d' ' -f1)
 
   NAMESPACE=$1
   USERNAME=$2
 
-  kubectl exec testpod -- env UU="$USERNAME" NN="$NAMESPACE" JJ="$SPARK_EXAMPLES_JAR_NAME" \
+  SPARK_IMAGE='ghcr.io/canonical/charmed-spark:3.4.0-22.04_edge'
```

Comment:
**niptick** Why not factoring this out in a single place at the top of the file? I can see people changing this only in one place and not another...

Reply:
Done